### PR TITLE
Adding github actions workflow

### LINKED ISSUE #56: Move CI to GH Actions
Also:

- enable dependabot
- configure automerge

### PR DESCRIPTION
**Pull request by inigo-garcia**
Migrating CI to GH Actions and including dependabot. It runs all CI tests on Node v12 and v14. 

Closes #56

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: daily
```
Comment (simoneb) [inline .github/dependabot.yml:6]: include updating of github actions too

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+name: CI workflow
```
Comment (simoneb) [inline .github/workflows/ci.yml:1]: name it just ci for convention and reflect the change in the readme file

**Review by simoneb — COMMENTED** [review]
I believe the new action won't run until you mark this as ready for review

File: .github/dependabot.yml (hunk 1 above)
Comment (inigo-garcia) [reply .github/dependabot.yml:11]: Thanks, I included also the open pull request limit to 10

File: .github/workflows/ci.yml (hunk 2 above)
Comment (inigo-garcia) [reply .github/workflows/ci.yml]: Changed to `ci`

**Pull request closed without merging**